.github/workflows/centos8-wheel-tester.yaml

```diff
@@ -53,9 +53,11 @@ jobs:
               from cryptography.hazmat.backends.openssl.backend import backend
               print('Loaded: ' + backend.openssl_version_text())
               print('Linked Against: ' + backend._ffi.string(backend._lib.OPENSSL_VERSION_TEXT).decode('ascii'))
+          - PIP_NAME: sqlalchemy
+            PKG_TEST: import sqlalchemy
     name: "Centos8 for ${{ matrix.WHEEL.PIP_NAME }}"
     steps:
-      - run: yum install -y python3-pip
+      - run: yum install -y python3-pip gcc
```

Comment:
People trying to use python packages via pip should ultimately not have to have a development environment installed to be able to use Arm64 packages.  Therefore I'm placing the bar at: has to work only with a simple `pip install` same as x86.

Reply:
@geoffreyblake I agree with you, removed development environment.